diff --git a/vim/bundle/code_complete/plugin/code_complete.vim b/plugin/code_complete.vim
index 01fef67..9b65d4b 100644
--- a/vim/bundle/code_complete/plugin/code_complete.vim
+++ b/vim/bundle/code_complete/plugin/code_complete.vim
@@ -226,7 +226,7 @@ function! GetFileName()
     let filename=expand("%:t")
     let filename=toupper(filename)
     let _name=substitute(filename,'\.','_',"g")
-    "let _name="__"._name."__"
+    let _name=_name."_"
     return _name
 endfunction
 
@@ -245,11 +245,11 @@ let g:template = {}
 let g:template['c'] = {}
 let g:template['c']['cc'] = "/*  */\<left>\<left>\<left>"
 let g:template['c']['cd'] = "/**<  */\<left>\<left>\<left>"
-let g:template['c']['de'] = "#define     "
-let g:template['c']['in'] = "#include    \"\"\<left>"
-let g:template['c']['is'] = "#include  <>\<left>"
-let g:template['c']['ff'] = "#ifndef  \<c-r>=GetFileName()\<cr>\<CR>#define  \<c-r>=GetFileName()\<cr>".
-            \repeat("\<cr>",5)."#endif  /*\<c-r>=GetFileName()\<cr>*/".repeat("\<up>",3)
+let g:template['c']['de'] = "#define "
+let g:template['c']['in'] = "#include \"\"\<left>"
+let g:template['c']['is'] = "#include <>\<left>"
+let g:template['c']['ff'] = "#ifndef \<c-r>=GetFileName()\<cr>\<CR>#define \<c-r>=GetFileName()\<cr>".
+            \repeat("\<cr>",4)."#endif /* \<c-r>=GetFileName()\<cr> */".repeat("\<up>",2)
 let g:template['c']['for'] = "for( ".g:rs."...".g:re." ; ".g:rs."...".g:re." ; ".g:rs."...".g:re." )\<cr>{\<cr>".
             \g:rs."...".g:re."\<cr>}\<cr>"
 let g:template['c']['main'] = "int main(int argc, char \*argv\[\])\<cr>{\<cr>".g:rs."...".g:re."\<cr>}"
